src/jdk.javadoc/share/classes/jdk/javadoc/internal/doclets/toolkit/builders/SerializedFormBuilder.java

```diff
@@ -263,12 +264,12 @@ protected void buildClassContent(Content target) throws DocletException {
      */
     protected void buildSerializableMethods(Content target) throws DocletException {
         Content serializableMethodsHeader = methodWriter.getSerializableMethodsHeader();
-        SortedSet<ExecutableElement> members = utils.serializationMethods(currentTypeElement);
-        if (!members.isEmpty()) {
-            for (ExecutableElement member : members) {
-                currentMember = member;
+        Iterator<ExecutableElement> members = utils.serializationMethods(currentTypeElement).iterator();
+        if (members.hasNext()) {
+            while (members.hasNext()) {
```

Comment:
This double `hasNext` looks odd. I understand why you are doing it, so I'm not saying its wrong,
but to my eyes it would be nicer to get the `members` set and check `isEmpty` as before, while
keeping the use of the iterator within the `then` block.

Reply:
I'm glad you asked. Unlike that in SerializedFormBuilder.java:402, here, the first `hasNext` can be dropped. It was useless before, it is useless now. I left it initially, because it was unrelated.